frontend/src/pages/Users/Users.js

```diff
@@ -36,11 +36,24 @@ const styles = {
   }
 };
 const Users = props => {
-  const { tabIndex, setTabIndex, showDashboardDialog, allowedIntents, isDataLoading } = props;
+  const {
+    tabIndex,
+    setTabIndex,
+    showDashboardDialog,
+    allowedIntents,
+    isDataLoading,
+    enabledUsers,
+    disabledUsers
+  } = props;
   const isCreateButtonDisabled =
-    tabIndex === 0 ? !allowedIntents.includes("global.createUser") : !allowedIntents.includes("global.createGroup");
+    tabIndex === 0
```

Comment:
Only use conditional operator for one condition

Reply:
done